### PR TITLE
Fixes issue #182 and adds a unit test to check this condition.

### PR DESCRIPTION
**Pull request by RickSBrown**
Looks like the intent was for `decimal` and `number` to be validated the same way instead of `decimal` and `integer`.

### CODE REVIEW

**Comment by RickSBrown** [comment]
@alice PTAL

File: src/js/AccessibilityUtils.js [hunk 1]
```diff
@@ -654,7 +654,7 @@ axs.utils.getRoles = function(element, implicit) {
             roleObject.valid = result.valid = true;
         } else {
             roleObject.valid = false;
-            
+
```
Comment (alice) [inline src/js/AccessibilityUtils.js]: Weird that there was a bunch of spaces here before; can we remove the empty line as well?

File: src/js/AccessibilityUtils.js [hunk 2]
```diff
@@ -720,11 +719,13 @@ axs.utils.getAriaPropertyValue = function(propertyName, value, element) {
...
     case "number":
         var validNumber = axs.utils.isValidNumber(value);
-        if (validNumber.valid) {
-            result.valid = true;
-            result.value = validNumber.value;
+        if (!validNumber.valid) {
+            result.valid = false;
+            result.reason = validNumber.reason;
+            return result;
         }
```
Comment (alice) [inline src/js/AccessibilityUtils.js]: It also seems weird that we don't have a return for the valid case here (undoubtedly my fault) - mind adding one?

**Comment by alice** [comment]
:+1: once comments addressed. Nice work!

File: src/js/AccessibilityUtils.js (hunk 2 above)
Comment (RickSBrown) [inline src/js/AccessibilityUtils.js]: Hehe - I was scared to touch it - I'll fix it up :)

**Pull request merged**